1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers = numbers.sort((a, b) => a - b); // complete this statement
```

Comment:
This works!
A tip to simplify it even more - `arr.sort()` will do a simple ascending sort without needing to pass it a function.